src/services/Staffing.ts

```diff
@@ -97,71 +98,84 @@ export class Staffing {
+        return QueryString.stringify(params);
+      },
+    });
     if (!Staffing.httpInitialized) {
       Staffing.httpInitialized = true;
-      // Add a response interceptor
-      Staffing._http.interceptors.response.use(
-        (response: AxiosResponse) => {
-          // Tracking
-          const timing: { start: number; url: string } = (response.config as any)._timing || undefined;
-          if (Staffing.trackingCallback && timing) {
-            Staffing.trackingCallback(timing.url, new Date().getTime() - timing.start);
-          }
-          return response;
-        },
-        async (error: AxiosError) => {
-          // Tracking
-          const errorResponse: AxiosResponse = error.response;
-          const timing: { start: number; url: string } = (errorResponse.config as any)._timing || {};
-          if (Staffing.trackingCallback && timing) {
-            Staffing.trackingCallback(timing.url, new Date().getTime() - timing.start);
-          }
-          // Check if Unauthorized Error
-          if (error.response.status === 401) {
-            Staffing.unauthorized.next(error);
-          }
-          return Promise.reject(error);
-        },
-      );
-      // Add a request interceptor
-      Staffing._http.interceptors.request.use((config: AxiosRequestConfig) => {
-        // Set a timing config so that we can track request times
-        (config as any)._timing = {
-          start: new Date().getTime(),
-          url: (config.url || '').replace(config.baseURL || '', '').split('?')[0],
-        };
-        return config;
-      });
     }
+    // Add a response interceptor
+    instance.interceptors.response.use(
+      (response: AxiosResponse) => {
+        // Tracking
+        const timing: { start: number; url: string } = (response.config as any)._timing || undefined;
+        if (Staffing.trackingCallback && timing) {
+          Staffing.trackingCallback(timing.url, new Date().getTime() - timing.start);
+        }
+        return response;
+      },
+      async (error: AxiosError) => {
+        // Tracking
+        const errorResponse: AxiosResponse = error.response;
+        const timing: { start: number; url: string } = (errorResponse.config as any)._timing || {};
+        if (Staffing.trackingCallback && timing) {
+          Staffing.trackingCallback(timing.url, new Date().getTime() - timing.start);
+        }
+        // Check if Unauthorized Error
+        if (error.response.status === 401) {
+          Staffing.unauthorized.next(error);
+        }
+        return Promise.reject(error);
+      },
+    );
+    // Add a request interceptor
+    instance.interceptors.request.use((config: AxiosRequestConfig) => {
+      // Set a timing config so that we can track request times
+      (config as any)._timing = {
+        start: new Date().getTime(),
+        url: (config.url || '').replace(config.baseURL || '', '').split('?')[0],
+      };
+      config.headers.uniqueCallId = uuid();
+      if (routeUrl !== '') {
```

Comment:
from how this is being used in novo, i'm thinking it makes more sense to call this a routeUUID or a routeIdentifier or something else? since this value isn't actually a URL is it

Reply:
changed it to be called `callingIdentifier`